src/smartHandler.ts

```diff
@@ -167,8 +167,10 @@ export class SMARTHandler implements Authorization {
 
         if (fhirUserClaim && usableScopes.some((scope) => scope.startsWith('user/'))) {
             userIdentity.fhirUserObject = getFhirUser(fhirUserClaim);
-            userIdentity.patientOrgs = [];
-            if (patientOrgsClaim && Array.isArray(patientOrgsClaim)) {
```

Comment:
There are unit tests that a FHIR user has both user and system scope. In that case patientOrgsClaim is not required. (because the user has system scope).
But the above code requires patientOrgsClaim, otherwise it throws an error. Because it finds the user has user scope.

I think we may separate scopes and patientOrgsClaim. Let me know if it is okay to do so.

Reply:
I can't think of a real scenario where a client would have both scopes. For those tests can we just have an empty patientOrgsClaim?

Reply:
sure